test/urlencoded.js

```diff
@@ -177,7 +177,7 @@ describe('bodyParser.urlencoded()', function () {
       it('should parse deep object', function (done) {
         var str = 'foo'
 
-        for (var i = 0; i < 500; i++) {
+        for (var i = 0; i < 250; i++) {
```

Comment:
Looks like this is no longer a needed change with current ChakraCore (or, at least with current `master` and ChakraCore).